manifests/m/Microsoft/Edge/123.0.2420.53/Microsoft.Edge.installer.yaml

```
@@ -0,0 +1,42 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.Edge
+PackageVersion: 123.0.2420.53
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: msi
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- msedge
+Protocols:
+- http
+- https
+FileExtensions:
+- crx
+- htm
+- html
+- pdf
+- url
+AppsAndFeaturesEntries:
+- UpgradeCode: '{883C2625-37F7-357F-A0F4-DFAF391B2B9C}'
+Installers:
+- Architecture: x86
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/b16f1a1a-0ecb-4566-8d28-f4460b7de5c8/MicrosoftEdgeEnterpriseX86.msi
+  ProductCode: '{7DE886D9-A231-3D80-92FE-A6B6202BBA9B}'

```

```suggestion
  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/b16f1a1a-0ecb-4566-8d28-f4460b7de5c8/MicrosoftEdgeEnterpriseX86.msi
  ProductCode: '{7DE886D9-A231-3D80-92FE-A6B6202BBA9B}'
```
Hi @PckgrBot,

Could an `InstallerSha256` be added?